scripts/notifications.py

```diff
@@ -48,21 +93,18 @@ def cli(debug):
 @click.option(
     "--subject",
     help="Subject of release announcement message."
-         " May be omitted if create_release is used as that"
-         " generates a subject.")
+         " May be omitted if create_release is used.")
 @click.option(
     "--body",
-    help="Contents of release announcement message"
-         " May be omitted if generate_release_notes is used.")
+    help="Body of release announcement message."
```

Comment:
Please add the output of [usage](https://github.com/rcbops/rpc-gating/blob/master/scripts/release.py#L174) to [release.yml](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/release.yml#L71) so that the help text is in sync with the script.

Reply:
Done.